img/m/mulesoft.com.svg

```diff
@@ -0,0 +1 @@
+<svg viewBox="0 0 50 50" xmlns="http://www.w3.org/2000/svg"><path d="M21.9 42.3c-11.1 0-20.2-9-20.2-20.1S10.7 2 21.8 2 42 11 42 22.1c.1 11.1-8.9 20.2-20.1 20.2zm0-42.3C9.7.1-.1 9.9 0 22.1s9.9 22 22.1 21.9 22-9.9 21.9-22.1c0-5.8-2.4-11.4-6.5-15.5S27.7 0 21.9 0z" fill="#00a1df"/><path d="M16.1 32.2c-3.6-1.8-5.9-5.6-5.8-9.6 0-2.2.6-4.3 1.8-6.1l7.5 11.4h4.5l7.5-11.4c1.2 1.8 1.8 3.9 1.8 6.1 0 3.8-1.9 7.3-5.1 9.3l1.4 5.6c8.5-4.5 11.7-15.1 7.2-23.6-1.5-2.8-3.7-5.1-6.3-6.7l-8.7 13.6-8.6-13.7C5.1 12 2.4 22.7 7.3 31c1.7 2.9 4.3 5.2 7.4 6.7 0 .1 1.4-5.5 1.4-5.5z" fill="#00a1df" fill-rule="evenodd" clip-rule="evenodd"/></svg>
```

Comment:
I would like to ask if there are any requirements for this change in SVG? Because I use the MinfySVG extension in VSCode to compress it. If you can tell me the details, I will pay more attention to it in the subsequent submissions!

Reply:
The icon in the SVG you provided isn't centered and contains padding. That would make the icon look smaller than all the other icons and so I realigned it.

Regarding a requirement, the [guidelines](https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md#guidelines) requests that you minify any SVGs. Since many images are served to users, optimized images have a great effect on the bandwidth used to view the site.
I personally use [svgomg](https://jakearchibald.github.io/svgomg/) to minify SVGs with all options except _Remove xmlns_ turned on. The SVG test also provides useful information about unnecessary attributes, like in this case that `fill-rule` and `clip-rule` are used but not required.